app/assets/stylesheets/components/_containers.scss

```diff
@@ -0,0 +1,5 @@
+.container-for-navbar{
+  margin-top: 80px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
```

Comment:
For now there is only one container, so maybe rename it `_container.scss`
The `margin-bottom: 80px;` of `_card.scss` on class `.cards` may go on the general container, so the contact page can benefit it ?